hack/boilerplate/boilerplate.py

```diff
@@ -100,7 +100,8 @@ def file_passes(filename, refs, regexs):
 def file_extension(filename):
     return os.path.splitext(filename)[1].split(".")[-1].lower()
 
-skipped_dirs = ['Godeps', 'third_party', '_gopath', '_output', '.git', 'cluster/env.sh']
+skipped_dirs = ['Godeps', 'third_party', '_gopath', '_output', '.git', 'cluster/env.sh', "test/e2e/generated/bindata.go"]
```

Comment:
we could iff we ever added more generated components, but for now, only  `bindata.go` is the only file which i know of which shouldnt require boilerplate.....